commit 8e11ec10b769ee4de04159a05d5e701a712694cc
Date:   Fri Nov 10 15:56:03 2023 +0530

    Revert 'Add missing dependency (#1414)'
    
    The llbuild executable is not a dependency of this library.

diff --git a/swift-driver/Package.swift b/swift-driver/Package.swift
index f09f530d..0e1c989c 100644
--- a/swift-driver/Package.swift
+++ b/swift-driver/Package.swift
@@ -145,7 +145,6 @@ if ProcessInfo.processInfo.environment["SWIFT_DRIVER_LLBUILD_FWK"] == nil {
         ]
         package.targets.first(where: { $0.name == "SwiftDriverExecution" })!.dependencies += [
             .product(name: "llbuildSwift", package: "swift-llbuild"),
-            .product(name: "llbuild", package: "swift-llbuild"),
         ]
     } else {
         // In Swift CI, use a local path to llbuild to interoperate with tools
@@ -155,7 +154,6 @@ if ProcessInfo.processInfo.environment["SWIFT_DRIVER_LLBUILD_FWK"] == nil {
         ]
         package.targets.first(where: { $0.name == "SwiftDriverExecution" })!.dependencies += [
             .product(name: "llbuildSwift", package: "llbuild"),
-            .product(name: "llbuild", package: "llbuild"),
         ]
     }
 }
